src/components/EventList/EventList.tsx

```diff
@@ -42,6 +39,58 @@ const EventList = () => {
   const [searchQuery, setSearchQuery] = useState("");
   const [displayedEvents, setDisplayedEvents] = useState<Event[]>([]);
 
+  // Sample event data
```

Comment:
@arunimaChintu Why have you added this?

Reply:
Just to show the functionality of searching by tags